platformio.ini

```diff
@@ -12,19 +12,25 @@
 src_dir = firmware/esp32_ttgo
 data_dir = web/static
 
-[env:ttgo-lora32-v1]
+[env]
 platform = espressif32
 board = ttgo-lora32-v1
 framework = arduino
-upload_port = /dev/ttyUSB0
-monitor_port = /dev/ttyUSB0
+upload_port = /dev/cu.SLAB_USBtoUART
+monitor_port = /dev/cu.SLAB_USBtoUART
 monitor_speed = 115200
-
+build_flags =
+    ; -DTTGO_LORA_V1
+    ; -DTTGO_LORA_V2
+    ; -DTTGO_TBEAM
```

Comment:
I'm thinking of moving these into different build envs. For example, we could have something like this,
```
[env:ttgo-lora32-v1]
board = ttgo-lora32-v1
build_flags = -DTTGO_LORA_V1

[env:ttgo-lora32-v2]
board = ttgo-lora32-v1
build_flags = -DTTGO_LORA_V2

[env:ttgo-lora32-tbeam]
board = ttgo-lora32-v1
build_flags = -DTTGO_TBEAM
```
Everything else would be in the global `[env]` and we could set `ttgo-lora32-v2` as the default env, but then you could easily compile for different boards by running something like,
`pio run -e ttgo-lora-tbeam`

Reply:
Yeah this is actually what I'm doing locally, I just didn't commit it. I'll go ahead and do that.